src/components/ListItem.js

```diff
@@ -18,22 +18,6 @@ const spanTarget = {
   canDrop(props, monitor) {
     // You can disallow drop based on props or item
```

Comment:
@adamjarling  Why are we passing these 2 params if we are not using them? I'm asking this, just for me to understand the code.

Reply:
I realized that, this might be an in-built function from the DnD library after I made the comment. So I merged this anyway. Thanks for the explanation :+1: